packages/app/src/store/runner-ui-store.ts

```diff
@@ -22,6 +24,8 @@ export const useRunnerUiStore = defineStore({
       showChooseExternalEditorModal: false,
       autoScrollingEnabled: true,
       isSpecsListOpen: true,
+      specListWidth: 280,
```

Comment:
We have the numberes 280 and 320 in multiple places in this PR alone (just do cmd+F 320 and you'll see), can we consolidate them in a single place, maybe on an object, then import and use it everywhere? Otherwise if we change these numbers, we will need to remember to update it in many places.

Maybe

```ts
export const runnerConstants = {
  specListWidth: 280

  // ...
}
```

Then use it 

- https://github.com/cypress-io/cypress/pull/19303/files#diff-180e851fe76714c228c2c2156248463c450d0a8f2de324776e4139b11d0ad458R148
- https://github.com/cypress-io/cypress/pull/19303/files#diff-efe9214a23f25db2926aabc22f8bd169e81e77652428eceb1c195a0d6fb05161R77

Reply:
Done - except in one assertion where 280 was a coincidence, it didn't matter that panel 2 matched the default width of panel 1 at that moment. I updated the movement and the assertion to different values.